src/zarrzipan/cli.py

```diff
@@ -0,0 +1,335 @@
+                if n > output_files_n:
+                    output_files_n = n
+            except ValueError:
+                # For when the extracted group might not be a valid integer
+                continue
+
+    return output_dir / f'output{output_files_n + 1}.json'
+
+
+def _fetch_array(data_dir: Path, name: str, href: str) -> Path:
+    parts = urlsplit(href)
+    filename = parts.path.split('/')[-1]
+
+    filepath = data_dir / name / filename
+    filepath.parent.mkdir(parents=True, exist_ok=True)
+
+    if not filepath.exists():
+        click.echo(f'Fetching {href}')
+
+        store = HTTPStore(f'{parts.scheme}://{parts.netloc}')
+        resp = obs.get(store, parts.path)
+
+        with filepath.open('wb') as f:
+            for chunk in resp:
+                f.write(chunk)
+
+    return filepath
+
+
+def _create_table(results: list[dict[str, Any]]) -> Table:
+    """Create a rich table matching the markdown table"""
+    # Define the desired column order and their corresponding keys in the JSON data
+    desired_columns = [
+        ('Dataset Name', 'dataset_name'),
+        ('Pipeline Name', 'pipeline_name'),
+        ('Chunk Shape', 'chunk_shape'),
+        ('Iterations', 'iterations'),
+        ('Compression Ratio', 'size_ratio'),
+        ('Space Saving', 'size_space_saving'),
+        ('Avg Compress Time (ms)', 'time_compress_avg_ms'),
+        ('Avg Decompress Time (ms)', 'time_decompress_avg_ms'),
+        ('Lossiness (MAE)', 'lossiness_mae'),
+    ]
+
+    mean_compression_ratio = np.mean([item['size_ratio'] for item in results])
+
+    # Helper function to format values
+    def format_value(key, value):
+        if key == 'size_ratio':
+            color = 'cyan'
+            if value > mean_compression_ratio + 1:
+                color = 'green'
+            elif value < max(mean_compression_ratio - 1, 1):
+                color = 'red'
+            return Text(f'{value:.2f}x', style=color)
+        if key == 'lossiness_mae':
+            return f'{value:.4f}'
+        if key == 'chunk_shape':
+            return str(value).replace('[', '(').replace(']', ')')
+        if isinstance(value, float):
+            return f'{value:.2f}'
+        return str(value)
+
+    # Create a rich table
+    table = Table(title='Compression Benchmark Results')
+
+    # Add columns to the table
+    for header_name, _ in desired_columns:
+        table.add_column(header_name, justify='right', style='cyan', no_wrap=False)
+
+    # Add rows to the table
+    for data_item in results:
+        row_values = [format_value(key, data_item[key]) for _, key in desired_columns]
+        table.add_row(*row_values)
+
+    return table
+
+
+@click.group()
+@click.option('--verbose', '-v', is_flag=True, help='Print more output.')
+@click.pass_context
+def cli(ctx, verbose):
+    ctx.ensure_object(dict)
+    ctx.obj['v'] = verbose
+
+
+@cli.command()
+@click.pass_context
+@click.option(
+    '--config-file',
+    '-f',
+    default='config.yaml',
```

Comment:
Seems like this is a pretty generic name that could conflict. It might be better to name this zarrzipan.yml or something more unique.